src/mono/CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   set(HOST_OSX 1)
   set(PTHREAD_POINTER_ID 1)
   set(USE_MACH_SEMA 1)
-  if(CMAKE_SYSTEM_VARIANT STREQUAL "MacCatalyst")
+  if(CMAKE_SYSTEM_VARIANT STREQUAL "maccatalyst")
     set(HOST_MACCAT 1)
   endif()
 elseif(CMAKE_SYSTEM_NAME STREQUAL "iOS" OR CMAKE_SYSTEM_NAME STREQUAL "tvOS")
```

Comment:
can you please fix this as well, by introducing `CLR_CMAKE_HOST_OS` a few lines above? longer term I'd like to unify the separate Mono cmake files with configureplatform.cmake